docs/superpowers/plans/2026-05-03-opensearch-eks-reference-deployment.md

```
@@ -0,0 +1,842 @@
+    dask_worker -->|SigV4 ingest| aoss
+    pod_identity[EKS Pod Identity<br/>association] -.maps SA to.-> iam[IAM role<br/>aoss:APIAccessAll]
+    iam -.assumed by.-> backend
+    iam -.assumed by.-> dask_worker
+    aoss_dap[AOSS data access policy] -.grants index ops.-> iam
+```
+
+The backend pod and every Dask worker assume the same IAM role through the EKS Pod Identity
+association on the `aiq-backend` service account. Each Dask worker constructs its own OpenSearch
+client, so SigV4 signing happens in the worker's process — no signer state is serialized across
+the cluster.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)